### ISSUE TITLE
Build on cygwin with openssl support

### ISSUE THREAD

**Issue by ralcini** [esnet/iperf#983]
Unable to build from scratch iperf 3.7 on cygwin platform with ssl support enabled (installing libssl-devel that obsoletes libopenssl-devel)

* Actual Behavior
make failed with
```
$ make
Making all in src
make[1]: ingresso nella directory «/home/rober/iperf/src»
make  all-am
make[2]: ingresso nella directory «/home/rober/iperf/src»
gcc -DHAVE_CONFIG_H -I.    -g -g -O2 -Wall -MT iperf3-main.o -MD -MP -MF .deps/iperf3-main.Tpo -c -o iperf3-main.o `test -f 'main.c' || echo './'`main.c
mv -f .deps/iperf3-main.Tpo .deps/iperf3-main.Po
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.     -g -O2 -Wall -MT cjson.lo -MD -MP -MF .deps/cjson.Tpo -c -o cjson.lo cjson.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -g -O2 -Wall -MT cjson.lo -MD -MP -MF .deps/cjson.Tpo -c cjson.c  -DDLL_EXPORT -DPIC -o .libs/cjson.o
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -g -O2 -Wall -MT cjson.lo -MD -MP -MF .deps/cjson.Tpo -c cjson.c -o cjson.o >/dev/null 2>&1
mv -f .deps/cjson.Tpo .deps/cjson.Plo
/bin/sh ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I.     -g -O2 -Wall -MT iperf_api.lo -MD -MP -MF .deps/iperf_api.Tpo -c -o iperf_api.lo iperf_api.c
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -g -O2 -Wall -MT iperf_api.lo -MD -MP -MF .deps/iperf_api.Tpo -c iperf_api.c  -DDLL_EXPORT -DPIC -o .libs/iperf_api.o
In file included from /usr/include/w32api/Windows.h:95,
                 from iperf_api.c:71:
/usr/include/openssl/ossl_typ.h:126:29: error: expected ‘)’ before numeric constant
  126 | typedef struct X509_name_st X509_NAME;
      |                             ^~~~~~~~~
/usr/include/openssl/ossl_typ.h:167:33: error: expected ‘)’ before numeric constant
  167 | typedef struct ocsp_response_st OCSP_RESPONSE;
      |                                 ^~~~~~~~~~~~~
make[2]: *** [Makefile:936: iperf_api.lo] Errore 1
make[2]: uscita dalla directory «/home/rober/iperf/src»
make[1]: *** [Makefile:697: all] Errore 2
make[1]: uscita dalla directory «/home/rober/iperf/src»
make: *** [Makefile:387: all-recursive] Errore 1
```

**Comment by ralcini** [comment]
I'm working on that in order to solve the problem reported some time ago here: https://www.neowin.net/forum/topic/1234695-iperf-37-windows-build/?do=findComment&comment=598301304

one year ago..

**Comment by ralcini** [comment]
fixed in PR #984

Some useful references: https://github.com/VirusTotal/yara/issues/880
